### DISCUSSION TITLE
How can I enable other users to use the playground?

### DISCUSSION THREAD

**Discussion by ai2ys** [TabbyML/tabby#1307] [Q&A] [answered]
Hi,
I upgraded to TabbyML version 0.7.0.
Earlier it was possible to access the playground just using the link. Now there is a login. I created an admin user on my first login.
But how can I enable other to be able to use it? I added their email addresses in "Team Mangement" but no mail client is configured, so probably no email will be send. 
What has to get setup, so be able to invite team members? Where can information be found on this topic?

**Comment by wsxiaoys** [comment, marked as answer]
You can copy the invitation link and share it with your team members. They will be able to register using the invitation link.

Recording: https://jam.dev/c/a62c7305-48b3-4baa-8222-872b128ba336

**Reply by ai2ys** [reply]
The copy button is not working. When I click it the clipboard is either empty or contains the content from the previous CTRL+C.
I am using Edge browser, but also tried Chrome and FireFox. When clicking on the copy button nothing happens, also the green checkbox from the video does not appear.

**Reply by wsxiaoys** [reply]
could you provide a recording with jam.dev?

**Reply by wsxiaoys** [reply]
Just realized the fix we added for non-secured context is not included in 0.7.0: https://github.com/TabbyML/tabby/commit/3ca9049325b05cc405bfc0958d84f1d676f8ab02

Could you give [0.8.0-rc.0](https://github.com/TabbyML/tabby/releases/tag/v0.8.0-rc.0) a try?

**Reply by ai2ys** [reply]
Unfortunately, I cannot share a video using jam.dev.
But I could try `0.8.0-rc.0`

**Reply by ai2ys** [reply]
In version `0.8.0-rc.0` the copy button for the invitation links work. Thank you.

BTW the copy button for code snippets in the playground does also not work for version `0.7.0`.

**Reply by ai2ys** [reply]
I have an additional question. How can the password of the admin get reset in case it gets forgotten?

**Reply by wsxiaoys** [reply]
> I have an additional question. How can the password of the admin get reset in case it gets forgotten?

It's not something we support for now, as email integration is not yet complete. We will add password reset functionality once it is implemented. (part of 1.0 roadmap)

**Comment by ai2ys** [comment]
> You can copy the invitation link and share it with your team members. They will be able to register using the invitation link.
> 
> Recording: https://jam.dev/c/a62c7305-48b3-4baa-8222-872b128ba336

Thank you for the information.
Is there any written documentation available on this?

**Comment by wsxiaoys** [comment]
We have not yet begun to document these enterprise features, such as user management. However, these tasks need to be completed before the 1.0 release.
